src/Hl7.Fhir.Base/FhirPath/Expressions/SymbolTableInit.cs

```diff
@@ -223,6 +224,23 @@ public static SymbolTable AddStandardFP(this SymbolTable t)
         }
 
 
+        internal static void AddBuiltinChildren(this SymbolTable table)
```

Comment:
What does this do?

Reply:
It avoids the `Wrap` for the `builtin.children` function (which is added by the `Add` method used before), since input and output are known to be `IEnumerable<ITypedElement>`.